source/main_cr.f90

```diff
@@ -1313,8 +1295,8 @@ end subroutine envarsv
 
```

Comment:
ok the if statement for identifying a particle which should have longitudinal variables initialised to 0,1, ... should be modified into something like:
`
if ( abs((ejfv(j)/nucm(j))/(e0f/nucm0)-one).lt.c1m15 .or. abs((ejv(j)/nucm(j))/(e0/nucm0)-one).lt.c1m15 ) then
`